examples/kubernetes/manifests/registry.yml

```diff
@@ -0,0 +1,60 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: registry
+  labels:
+    app: registry
+    tier: registry
+spec:
+  type: NodePort
+  # type: LoadBalancer
+  ports:
+  - port: 5000
+    targetPort: 5000
+  selector:
+    app: registry
+    tier: registry
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: registry
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: registry
+        tier: registry
+    spec:
+      containers:
+      - name: registry
+        image: library/registry:2.6
+        command: ["/bin/sh", "/etc/docker/registry/init"]
```

Comment:
Rather than having a custom startup script, would it make sense to move the configuration to either environment variables (docker registry supports all configuration via env), or a configmap?